dqc_us_rules/dqc_us_0009.py

```diff
@@ -13,22 +13,41 @@
 def fact_gt_other_fact(val):
     """
     Checks that the various fact pairs that are defined in the configuration
-    are of an appropriate relative value (lesser or greater than one or the other, as per the config).
+    are of an appropriate relative value (lesser or greater than one or the
+    other, as per the config).
+
+    :param val: val to to throw error from
+    :type val: val
     """
     for index, lesser, greater in _read_csv():
         for fact_group in _compare_facts(lesser, greater, val):
             lesser_fact = fact_group[lesser]
             greater_fact = fact_group[greater]
-            val.modelXbrl.error('{base_code}.{index_id}'.format(base_code=_CODE_NAME, index_id=index),
-                                messages.get_message(_CODE_NAME, index),
-                                modelObject=[lesser_fact, greater_fact],
-                                lesser_label=lesser_fact.concept.label(),
-                                greater_label=greater_fact.concept.label(),
-                                ruleVersion=_RULE_VERSION)
+            val.modelXbrl.error(
+                '{base_code}.{index_id}'.format(
+                    base_code=_CODE_NAME,
+                    index_id=index
+                ),
+                messages.get_message(_CODE_NAME, index),
+                modelObject=[lesser_fact, greater_fact],
+                lesser_label=lesser_fact.concept.label(),
+                greater_label=greater_fact.concept.label(),
+                ruleVersion=_RULE_VERSION
+            )
 
 
 def _read_csv():
-    path = os.path.join(os.path.dirname(__file__), 'resources/DQC_US_0009/dqc_us_9_config.csv')
+    """
+    Reads in information from dqc_us_config.csv and returns
+    it in a tuple
+
+    :rtype: tuple of index, lesser fact and greater fact
```

Comment:
SG: `:rtype: tuple (int, OBJECT, OBJECT)` where object is the ```:class:`~path.to.Object` ``` for those objects.